android/brave_java_resources.gni

```diff
@@ -147,6 +147,7 @@ brave_java_resources = [
   "java/res/drawable-night/restore_wallet.xml",
   "java/res/drawable-night/tablet_recent_tab_empty_state_illustration.xml",
   "java/res/drawable-night/tipping_background_success.xml",
+  "java/res/drawable-night/wallet_portfolio.xml",
```

Comment:
@simoarpe we may need to update the icon here once we have new tokens @aguscruiz is looking into. For now, let's keep it as is.

Reply:
Sure! I've created a GH issue to keep track of this: https://github.com/brave/brave-browser/issues/39423